src/ddscxx/include/org/eclipse/cyclonedds/sub/AnyDataReaderDelegate.hpp

```diff
@@ -164,29 +168,29 @@ class OMG_DDS_API AnyDataReaderDelegate : public org::eclipse::cyclonedds::core:
     void add_query(org::eclipse::cyclonedds::sub::QueryDelegate& query);
     void remove_query(org::eclipse::cyclonedds::sub::QueryDelegate& query);
 
-    inline void setCopyIn(org::eclipse::cyclonedds::topic::copyInFunction copyIn)
+    inline void setCopyIn(org::eclipse::cyclonedds::topic::copyInFunction _copyIn)
     {
-        this->copyIn = copyIn;
+        this->copyIn = _copyIn;
     }
```

Comment:
`setCopyIn` can probably be removed entirely in the `idlcxx` branch? Same for `setCopyOut`? @e-hndrks, we probably don't need them anymore right?

Reply:
Removed this function, since it is not used anywhere.